pvfactors/geometry/base.py

```diff
@@ -64,36 +64,44 @@ def coords_from_center_tilt_length(xy_center, tilt, length,
     ----------
     xy_center : tuple
         x, y coordinates of center point of desired linestring
-    tilt : float
-        surface tilt angle desired [deg]
+    tilt : float or np.ndarray
+        Surface tilt angles desired [deg]. Values should all be positive.
     length : float
         desired length of linestring [m]
-    surface_azimuth : float
-        Surface azimuth of PV surface [deg]
+    surface_azimuth : float or np.ndarray
+        Surface azimuth angles of PV surface [deg]
     axis_azimuth : float
         Axis azimuth of the PV surface, i.e. direction of axis of rotation
         [deg]
 
     Returns
     -------
     list
-        List of linestring coordinates obtained from inputs
+        List of linestring coordinates obtained from inputs (could be vectors)
+        in the form of [[x1, y1], [x2, y2]]
     """
-    is_pointing_right = ((surface_azimuth - axis_azimuth) % 360.) > 180.
-    if is_pointing_right:
-        # system should be rotated to the right: so negative rotation angle
-        rotation = tilt
-    else:
-        # need positive rotation angle for system to point to the left
-        rotation = - tilt
+    # PV row params
     x_center, y_center = xy_center
     radius = length / 2.
+    # Get rotation
+    rotation = get_rotation_from_tilt_azimuth(surface_azimuth, axis_azimuth,
+                                              tilt)
```

Comment:
the fn ``get_rotation_from_tilt_azimuth`` will be re-used in ``OrderedPVArray``.
The ``rotation`` output here could actually be returned by this ``coords_from_center_tilt_length()`` fn, but it would not really make sense...